src/utils/hact/raise-repair-form.js

```diff
@@ -31,17 +37,59 @@ export const buildRaiseRepairFormData = (formData) => {
         ],
       },
     ],
-    sitePropertyUnit: [
-      {
-        address: {
-          addressLine: [formData.shortAddress],
+    site: {
+      property: [
+        {
+          propertyReference: formData.propertyReference,
+          address: {
+            addressLine: [formData.shortAddress],
+          },
+          reference: [
+            {
+              id: formData.propertyReference,
+            },
+          ],
         },
+      ],
+    },
+    instructedBy: {
+      name: 'Hackney Housing',
+    },
+    assignedToPrimary: {
+      name: `Contractor ${formData.sorCodeContractorRef}`,
```

Comment:
This will have the name of the contractor when this exists in the api but for now sending up the reference code